jobs/rocoto/prep.sh

```diff
@@ -36,7 +36,7 @@ export COMOUT="$ROTDIR/$CDUMP.$PDY/$cyc/$COMPONENT"
 ###############################################################
 # If ROTDIR_DUMP=YES, copy dump files to rotdir 
 if [ $ROTDIR_DUMP = "YES" ]; then
-   $HOMEgfs/ush/getdump.sh $CDATE $CDUMP $DMPDIR/${CDUMP}${DUMP_SUFFIX}.${PDY}/${cyc} $COMOUT
+   $HOMEgfs/ush/getdump.sh $CDATE $CDUMP $DMPDIR/${CDUMP}${DUMP_SUFFIX}.${PDY}/${cyc}/${COMPONENT} $COMOUT
```

Comment:
Where does `prep.sh` get `$COMPONENT` from? `config.prep`?

Reply:
`prep.sh` has this nearer the top:
```
export COMPONENT=${COMPONENT:-atmos}
```
https://github.com/KateFriedman-NOAA/global-workflow/blob/feature/dev_v16_gdaatmos/jobs/rocoto/prep.sh#L31